packages/app/src/containers/SwatchesToolbar.tsx

```diff
@@ -94,10 +103,33 @@ export default function SwatchesToolbar() {
           >
             <PlusIcon />
           </Button>
+          <Spacer.Horizontal size={4} />
+          <Button
+            id="add-style"
+            onClick={() => {
```

Comment:
We should add a `useCallback` here. This is the function that would need `useCallback` most, since without it, the `Button` will always re-render